src/verify_cert.rs

```diff
@@ -485,18 +501,38 @@ impl ExtendedKeyUsage {
         let input = match (input, self) {
             (Some(input), _) => input,
             (None, Self::RequiredIfPresent(_)) => return Ok(()),
-            (None, Self::Required(_)) => return Err(Error::RequiredEkuNotFound),
+            (None, Self::Required(_)) => {
```

Comment:
codecov is flagging missing coverage for L505..512 - that seems worth having IMO.